src/test/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionaryIntegrationTest.java

```diff
@@ -35,6 +35,22 @@ public void testBasic() throws Exception {
         IntegrationTestSpec.assertEqualTextFiles(outputDict, new File(BASIC_FASTA.getAbsolutePath() + ".dict"));
     }
 
+    @Test
+    public void testNoOutput() throws Exception {
+        final File expectedDict = new File(TEST_DATA_DIR, "basic.dict");
```

Comment:
This is not an output file to create, but the expected output file for the tool. It is exists under `TEST_DATA_DIR`.